client-side/src/components/signUp/PasswordStrength.jsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { Box, LinearProgress, Typography } from '@mui/material';
+const PasswordStrengthMeter = ({ password }) => {
+  const PASSWORD_STRENGTH = {
```

Comment:
this enum should be in constants file.
and the messages should be more detailed, so the user will be able to understand his mistake

Reply:
the messages has not enough details